pcli/src/main.rs

```diff
@@ -78,6 +78,12 @@ enum Addr {
     },
 }
 
+#[derive(Debug, StructOpt)]
+enum Tx {
+    /// Send transaction to the node.
+    Send,
```

Comment:
This should probably eventually specify the amount, denom, and destination addresses, right?  (even if we're not currently using them for the dummy transactions?)

Reply:
maybe we want to allow multiple (amount, denom) pairs?

Reply:
I think initially supporting one spend makes sense (and then I guess under the hood we'd have the transaction builder create two spends in that case, one to the dest address, one with the change). The other thing we'd want as a required field is the fee (required field in the transaction builder).